### PR TITLE
Update CLI documentation

### PR DESCRIPTION
**Pull request by mattias-p**
## Purpose

This PR updates the end user documentation for CLI on how to get brief and full command line documentation.

## Context

This the sister PR of zonemaster/zonemaster-cli#389.

## Changes

* `docs/public/using/cli.md`

## How to test this PR

Run the commands in the update and make sure they match the description.

### CODE REVIEW

File: docs/public/using/cli.md [hunk 1]
```diff
@@ -112,12 +112,21 @@ docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6 --show-testcase --loc
...
+Run `zonemaster-cli --help` to get brief descriptions of a selection of the most
+important command line options.
 
-```
+```sh
 zonemaster-cli --help
 ```
 
+For complete reference documentation, see the manual page.
+This includes an exhaustive list of options and in-depth documentation each one,
```
Comment (marc-vanderwal) [inline docs/public/using/cli.md:123]:
```suggestion
This includes an exhaustive list of options and in-depth documentation for each one,
```

File: docs/public/development/cli.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Common Developer Actions
+
+### Generate man page
+
+To generate and view a development version of the a man page:
+
+```sh
+make manifypods
+man blib/man1/zonemaster-cli.1p
```
Comment (matsduf) [inline docs/public/development/cli.md:17]: On FreeBSD:
```
man blib/man1/zonemaster-cli.1
```

Isn't simpler to refer to `perldoc -oman zonemaster.cli`?

**Review by matsduf — CHANGES_REQUESTED** [review]
Besides the comment it looks fine.

**Comment by matsduf** [comment]
Besides, every directory in the public tree must have a `README.md` and all files in the public tree must be linked from `SUMMARY.md` (except `SUMMARY.md` and `book.toml`).

Maybe we should have a CI for this repository, though the link checking does not seem to be black or white.

File: docs/public/using/cli.md [hunk 3]
```diff
@@ -112,12 +112,21 @@ docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6 --show-testcase --loc
 zonemaster-cli zonemaster.net --no-ipv6 --show-testcase --locale=da_DK.UTF-8
 ```
 
-To see all available command line options, use the `--help` command.
+Run `zonemaster-cli --help` to get brief descriptions of a selection of the most
+important command line options.
```
Comment (matsduf) [inline docs/public/using/cli.md:116]: The command itself does not have to be repeated.

```suggestion
To get brief descriptions of a selection of the most important command line
options run
```

**Review by matsduf — COMMENTED** [review]
It looks good, just one small comment.

**Review by matsduf — APPROVED** [review]

**Pull request merged**